src/main.cpp

```diff
@@ -514,7 +514,8 @@ int main(int argc, char** argv) {
             std::make_unique<ExpandFilterTransformer>(), std::make_unique<HoistConditionsTransformer>(),
             std::make_unique<RamConditionalTransformer>(
                     []() -> bool { return std::stoi(Global::config().get("jobs")) > 1; },
-                    std::make_unique<ParallelTransformer>()));
+                    std::make_unique<ParallelTransformer>()),
+            std::make_unique<ReportIndexTransfomer>());
```

Comment:
I think OSX/clang tests on travis are failing because of this:
```
Undefined symbols for architecture x86_64:
  "souffle::RamIndexAnalysis::name", referenced from:
      souffle::RamIndexAnalysis* souffle::RamTranslationUnit::getAnalysis<souffle::RamIndexAnalysis>() const in main.o
      souffle::RamIndexAnalysis* souffle::RamTranslationUnit::getAnalysis<souffle::RamIndexAnalysis>() const in libsouffle.a(libsouffle_la-Synthesiser.o)
```
I don't see the problem immediately, since  the tests pass on other platforms.